### ISSUE TITLE
download timed out

### ISSUE THREAD

**Issue by David-Zhang-test** [renyunkang/yuque-exporter#3]
I have successfully logged in and all the book stacks are read, so all articles are listed without error.
But when it comes to download and export all books, it says download timed out therefore nothing is downloaded. I have no idea why this happen.
It's nothing about internet error since I have tested with and without proxy. Hope you can fix this bug and I'm also looking for solutions. If anything progress, I will update under this issue. Thanks
![image](https://github.com/renyunkang/yuque-exporter/assets/62581227/d3d46948-79ea-4c9f-8e1e-a8d0190c4e69)

**Comment by MarkLux** [comment]
same problem

**Comment by mtt0** [comment]
Any updates? Today yuque is broken... I want to migrate my notes too...

**Comment by keithMonster** [comment]
Need to modify ACCESSURL https://www.yuque.com/xxx -> xxx.
https://www.yuque.com/ has been spliced into the `downloadMardown` methods, so you only need to fill in the following xxx.

**Comment by VegetabllyDog** [comment]
> Need to modify ACCESSURL https://www.yuque.com/xxx -> xxx. https://www.yuque.com/ has been spliced into the `downloadMardown` methods, so you only need to fill in the following xxx.

it work for me, thank you ! 😄

**Comment by David-Zhang-test** [comment]
> Need to modify ACCESSURL https://www.yuque.com/xxx -> xxx. https://www.yuque.com/ has been spliced into the `downloadMardown` methods, so you only need to fill in the following xxx.

Yeah you're right. I'm busy focusing on other stuffs and forget this. After the ridiculous shutdown yesterday, it's time to export everything out of YuQue.

YuQue has changed its url rules...

**Comment by RoyRao2333** [comment]
@David-Zhang-test 

大佬你好，我今天使用这个脚本的时候，下载依然Timeout，代码修改如下：

<img width="1137" alt="image" src="https://github.com/renyunkang/yuque-exporter/assets/31413093/63449afe-44be-4c44-b21e-45f5da6fc844">

<img width="660" alt="image" src="https://github.com/renyunkang/yuque-exporter/assets/31413093/ce24b31f-2acd-4aad-a999-981bbcf5f4a4">

单独复制下载链接到另一个浏览器里去下载没有问题，在当前被Puppeteer启动的浏览器实例中进行下载就有问题

**Comment by RoyRao2333** [comment]
@dongchaoge 

I'm still getting timeout today：

<img width="1137" alt="image" src="https://github.com/renyunkang/yuque-exporter/assets/31413093/63449afe-44be-4c44-b21e-45f5da6fc844">

<img width="660" alt="image" src="https://github.com/renyunkang/yuque-exporter/assets/31413093/ce24b31f-2acd-4aad-a999-981bbcf5f4a4">

Copying the download link to another browser is fine, but downloading it in the current browser instance where Puppeteer is launched is problematic.

**Comment by renyunkang** [comment]
There are some updates, you can try the new version.

The download timeout may be caused by some special characters in the file name or other reasons.
